gravity-forms/gw-validate-current-password.php

```diff
@@ -0,0 +1,14 @@
+<?php
+/**
+ * Gravity Wiz // Gravity Forms // Validate Current Password
+ * http://gravitywiz.com/
+ */
+// update "123" to your form ID, update "6" to your Password field ID
+add_filter( 'gform_field_validation_1639_1', function( $result, $value ) {
```

Comment:
Update "1639" here to "123" and "1" to "4".

Reply:
Done